components/central-application-gateway/README.md

```diff
@@ -0,0 +1,109 @@
+
+### Start the Central Application Gateway
+
+To start the Central Application Gateway, run this command:
+
+```
+./applicationgateway
+```
+
+The Central Application Gateway has the following parameters:
+- **disableLegacyConnectivity** is the flag for disabling the default legacy and enabling the Compass mode. The default value is `false`.
+- **proxyPort** is the port that acts as a proxy for the calls from services and Functions to an external solution. The default port is `8080`.
+- **externalAPIPort** is the port that exposes the API allowing to check component status. The default port is `8081`.
+- **application** is the Application name used to write and read information about services. The default Application is `default-ec`.
+- **namespace** is the Namespace in which the Central Application Gateway is deployed. The default Namespace is `kyma-system`.
+- **requestTimeout** is the timeout for requests sent through the Central Application Gateway, expressed in seconds. The default value is `1`.
+- **skipVerify** is the flag for skipping the verification of certificates for the proxy targets. The default value is `false`.
+- **requestLogging** is the flag for logging incoming requests. The default value is `false`.
+- **proxyTimeout** is the timeout for requests sent through the proxy, expressed in seconds. The default value is `10`.
+- **proxyCacheTTL** is the time to live of the remote API information stored in the proxy cache, expressed in seconds. The default value is `120`.
+
+
+## API
+The Central Application Gateway exposes:
+- external API implementing health endpoint for liveness and readiness probes
+- internal API implementing proxy handler accessible via a service of ClusterIP type
+
+### Standalone mode
+In case  **disableLegacyConnectivity** is `false` the proxy API exposes the following endpoint:
+```{Application name}/{Service name}/{target API path}``` 
+
+For instance, if the user registered `cc-occ-commerce-webservices` service in `ec` application using Application Registry, they can send a request to the following url:
+```http://central-application-gateway:8080/ec/cc-occ-commerce-webservices/basesites```
+
+As a result Central Application Gateway performs the following steps:
+- looks for `cc-occ-commerce-webservices` service in `ec` Application CRD and extracts target URL path along with authentication configuration
+- modifies request to include authentication data
+- sends request to the following path `{target URL extracted from the Application CRD}/basesites`
+
+### Compass mode
+In case  **disableLegacyConnectivity** is `true` the proxy API exposes the following endpoint:
+```{Application name}/{API bundle name}/{API definition name}/{target api path}``` 
+
+For instance, if the user registered `cc-occ` API bundle with `commerce-webservices` API definition in `ec` application, they can send a request to the following url:
+```http://central-application-gateway:8082/ec/cc-occ/commerce-webservices/basesites``` 
```

Comment:
``````suggestion
For instance, if the user registered the `cc-occ` API bundle with the `commerce-webservices` API definition in the `ec` application, they can send a request to the following URL:
```bash
http://central-application-gateway:8082/ec/cc-occ/commerce-webservices/basesites
``` 
``````